ampl-executor/src/main/java/com/powsybl/ampl/executor/IAmplInputFile.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.ampl.executor;
+
+import com.powsybl.ampl.converter.AmplSubset;
+import com.powsybl.commons.util.StringToIntMapper;
+
+import java.io.InputStream;
+
+/**
+ * An additional file provided to an Ampl run.
+ * Useful when you need more information for your ampl model.
+ *
+ * @author Nicolas PIERRE <nicolas.pierre@artelys.com>
+ */
+public interface IAmplInputFile {
```

Comment:
A general comment. Even if quite common, we (in powsybl) have stated that interface should not start with I. Here you should call it AmplInputFile.